.gitignore

```diff
@@ -38,4 +38,7 @@ build/
 Debug/*
 Release/*
 ipch/
-.vs/
\ No newline at end of file
+.vs/
+*/.output
+*.zip
+/.output/*/*.lib
```

Comment:
I think all .output folder is now ignored (see line 2)

Reply:
Respite all this ignores I still see .lib files when committing ;/

Reply:
.lib files in output folder? can you show a screenshot of git/whatever you use?

Reply:
![obraz](https://github.com/cleolibrary/CLEO4/assets/11084446/70dcfa84-b5d5-4b80-8c4d-c2e224c1ad89)